app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java

```diff
@@ -63,11 +63,6 @@ public void describeTo(Description description) {
         };
     }
 
-    @Test
```

Comment:
Why was this test removed?

Reply:
I don't know, I had no reason to change anything in the MenuActivity, I added it back, good remark